.cargo/config.toml

```diff
@@ -0,0 +1,6 @@
+[build]
+target = "powerpc-unknown-eabi.json"
+rustflags = ["-C", "link-args=-lc"]
```

Comment:
Does the project build successfully if this line is removed? I think the ``c`` library is included in the ``build.rs`` file located in ``ogc-sys``, so this might not be needed.

Reply:
Yes, `rustflags = ["-C", "link-args=-lc"]` isn't needed anymore. I think it was needed previously. Removed.